src/displayScore.js

```diff
@@ -0,0 +1,16 @@
+import addScore from './addScore.js';
+
+const scoreList = document.querySelector('.score-list');
+
+const displayScores = async () => {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/eJRS7w29YVAaVnC0rjhT/scores');
+  const data = await response.json();
+  let dataArray = data.result;
+  dataArray = dataArray.sort((a, b) => b.score - a.score);
+  scoreList.innerHTML = '';
+  dataArray.forEach((score) => {
+    addScore(score.user, score.score);
+  });
```

Comment:
- [OPTIONAL] Great job @Lizdev-05 👍🏻 what if you use the JavaScript try and catch methods ? it's really simple:

const getScores = async () => {
  try{
    let response = await fetch(https:// ....);
        const data = await response.json();
        let dataArray = data.result;
       dataArray = dataArray.sort((a, b) => b.score - a.score);
       scoreList.innerHTML = '';
       dataArray.forEach((score) => {
         addScore(score.user, score.score);
       });
  } catch(error){
    console.log(error.message)
  }
};

- For more documentation you can check  [JavaScript try and catch](https://www.w3schools.com/js/js_errors.asp)